src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -1943,11 +2053,8 @@ public static void StopFeedback()
         /// Called when the vc client sends audio streaming for TTS feedback.
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
-        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
-        /// <privlevel>public</privlevel>
-        /// <feature>http://tizen.org/feature/speech.control</feature>
-        /// <feature>http://tizen.org/feature/microphone</feature>
-        /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
```

Comment:
invalid exception for event becuase event handler have not argument

Reply:
Removed ArgumentException